Products/ZenEvents/zentrap.py

```diff
@@ -620,12 +619,12 @@ def decodeSnmpv1(self, addr, pdu):
         # Look for the standard trap types and decode them without
         # relying on any MIBs being loaded.
         eventType = {
-            0: 'snmp_coldStart',
-            1: 'snmp_warmStart',
-            2: 'snmp_linkDown',
-            3: 'snmp_linkUp',
-            4: 'snmp_authenticationFailure',
-            5: 'snmp_egpNeighorLoss',
```

Comment:
this looks reasonable, but why not get rid of the whole event name map, and just set eventType to name if the oid2name() call returns a valid name?